### DISCUSSION TITLE
parse config error : proxy 0: unsupport proxy type: vless

### DISCUSSION THREAD

**Discussion by hao-888** [kkkgo/PaoPaoGateWay#104] [问和答]
我用vless弄了一个节点，导入后显示不支持，应该怎么设置

**Comment by hao-888** [comment]
![微信截图_20240620115734](https://github.com/kkkgo/PaoPaoGateWay/assets/64783863/f96d656c-d7b1-4b17-b12c-fafdec559b97)

**Comment by kkkgo** [comment]
使用docker定制镜像，替换mihomo内核。

**Comment by hao-888** [comment]
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/64783863/967fdb23-184e-4885-984d-d0d96db1b7aa)
想问一下，我应该下载哪一个版本，应该怎么重命名

**Reply by kkkgo** [reply]
解压重命名为clash。linux，amd64,compatible。

**Reply by kkkgo** [reply]
你截图的下载页面就有连接说“我该下载哪个文件”。https://github.com/MetaCubeX/mihomo/wiki/FAQ
